### PR TITLE
PLT-4534 Avoid applying themed hover background color on mobile

### PR DESCRIPTION
**Pull request by thomchop**
#### Summary
When applying theme styles, we were using the `centerChannelColor` as the background color for posts and modal list items when they user hovered on them. This was presumably only intended for true hovering when on desktop, but whenever the user touched these items on mobile the background color was being applied. This was most noticeable when scrolling through a posts list (in the channel view or RHS); every post the user touched had the background color displayed. As a form of haptic feedback it indicated to the user that by pressing (or long pressing) on a post they were supposed to be able to trigger some action, which isn't the case. This misleading styling was also present wherever there was a list of users in a modal (e.g. view team members, view/manage/add channel members, or new direct message).

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-4534

#### Checklist
- [ ] Added or updated unit tests (required for all new features)
- [ ] Added API documentation (required for all new APIs)
- [ ] All new/modified APIs include changes to the drivers
- [ ] Has enterprise changes (please link)
- [x] Has UI changes
- [ ] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/platform/blob/master/i18n/en.json) and [.../webapp/i18n/en.json](https://github.com/mattermost/platform/tree/master/webapp/i18n/en.json)) updates
- [ ] Touches critical sections of the codebase (auth, upgrade, etc.)

### CODE REVIEW

**Comment by jwilander** [comment]
@asaadmahmood can you take a quick look?

**Comment by asaadmahmood** [comment]
+1

File: webapp/utils/utils.jsx [hunk 1]
```diff
@@ -608,7 +608,8 @@ export function applyTheme(theme) {
...
         changeCss('.app__body .post.post--comment.other--root.current--user .post-comment, .app__body .more-modal__list .more-modal__row, .app__body .member-div:first-child, .app__body .member-div, .app__body .access-history__table .access__report, .app__body .activity-log__table', 'border-color:' + changeOpacity(theme.centerChannelColor, 0.1), 2);
         changeCss('@media(max-width: 1800px){.app__body .inner-wrap.move--left .post.post--comment.same--root', 'border-color:' + changeOpacity(theme.centerChannelColor, 0.07), 2);
-        changeCss('.app__body .post:hover, .app__body .post.post--hovered, .app__body .more-modal__list .more-modal__row:hover, .app__body .modal .settings-modal .settings-table .settings-content .section-min:hover', 'background:' + changeOpacity(theme.centerChannelColor, 0.08), 1);
+        changeCss('.app__body .post.post--hovered', 'background:' + changeOpacity(theme.centerChannelColor, 0.08), 1);
```
Comment (asaadmahmood) [inline webapp/utils/utils.jsx]: @thomchop You need to remove the closing bracket here in the media query for this to work.
Currently with your changes, the hover does not work for screens larger than 768px.

Comment (asaadmahmood) [inline webapp/utils/utils.jsx]: No worries. +1.

**Pull request merged**